docs/guide/types/box.md

```diff
@@ -92,6 +93,7 @@ If one of the axes does not match an axis in the chart, the box will take the en
 | `yMax` | Bottom edge of the box in units along the y axis.
 | `yMin` | Top edge of the box in units along the y axis.
 | `yScaleID` | ID of the Y scale to bind onto. If missing, the plugin will try to use the scale of the chart, configured as `'y'` axis. If more than one scale has been defined in the chart as `'y'` axis, the option is mandatory to select the right scale.
+| `z` | The `z` property determines the drawing stack level of the box annotation element. All visible elements will be drawn in crescent order of `z` option, with the same `drawTime` option.
```

Comment:
Don't think `cresent` is the correct wording, guess it should be `ascending`. Might be my lack of english but also when I putted it through translate it didn't give the results I would have thought.

Same comment for the other files

Reply:
Using Google it is translated correctly but I agree to change in `ascending`

Reply:
Changed using `ascending`

Reply:
aah yeah seems I read over it, it was at bottom of verbs, cuz all other references google thought it could be where about the moon 😅